<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>341</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="cardsLoaded">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateCardsButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>530</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Cards</string>
    </property>
   </widget>
   <widget class="QLabel" name="cardDisplayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input zone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cards in system</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="currentCardData">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current card text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeSelectedCardButton">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="templateButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate template</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSystemButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add system</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDamageButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add damage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFromPngButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from PNG sheets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBgImageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set background image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
